packages/marketplace/src/components/pages/settings/developer-settings-billing-tab/developer-settings-billing-tab.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+import { FlexContainerResponsive, Content, FlexContainerBasic } from '@reapit/elements'
+import { Tabs } from '../tabs'
+
+const DevelperSettingsBillingTabPage: React.FC<{}> = () => {
```

Comment:
you can remove `<{}>` it if there is no props `const DevelperSettingsBillingTabPage: React.FC = () => {`